Source/WebCore/rendering/svg/RenderSVGRoot.h

```diff
@@ -108,7 +108,6 @@ class RenderSVGRoot final : public RenderReplaced {
     bool nodeAtPoint(const HitTestRequest&, HitTestResult&, const HitTestLocation& locationInContainer, const LayoutPoint& accumulatedOffset, HitTestAction) final;
 
     LayoutRect overflowClipRect(const LayoutPoint& location, RenderFragmentContainer* = nullptr, OverlayScrollbarSizeRelevancy = IgnoreOverlayScrollbarSize, PaintPhase = PaintPhase::BlockBackground) const final;
-    LayoutRect clippedOverflowRect(const RenderLayerModelObject* repaintContainer, VisibleRectContext) const final;
```

Comment:
Why is it safe to remove this?

Reply:
Oh, the code is no identical to the RenderReplaced variant.